vignettes/indicesExistenceComparison.Rmd

```diff
@@ -206,7 +206,7 @@ df %>%
          index = factor(index, levels=c("p_direction", "p_MAP", "p_ROPE", "ROPE_89", "ROPE_95", "ROPE_full", "BF_log"))) %>%
   mutate(temp = as.factor(cut(sample_size, 3, labels = FALSE))) %>% 
   group_by(temp) %>% 
-  mutate(size_group = as.factor(round(mean(sample_size)))) %>% 
+  mutate(size_group = as.character(round(mean(sample_size)))) %>% 
```

Comment:
I'm, not sure if you need a factor for proper sorting in the plot, but as not all values of `sample_size` are present in all groups, `mutate()` coerces to character, anyway, so this avoids the warning.

If a factor is required, you need to specify _all_ available values from `sample_size` in the `levels`-argument, I guess (and hence, you need something like `all_levels <- unique(df$sample_size)` before the pipe-chain).